### PR TITLE
Feature To Develop edited 

### PR DESCRIPTION
**Pull request by manishmowall**
this body of feature to develop edited

### CODE REVIEW

**Comment by nitishm-webonise** [comment]
first comment made

**Comment by nitishm-webonise** [comment]
second comment made

**Review by manishmowall — COMMENTED** [review]
first review

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (manishmowall) [inline pom.xml:4]: first review comment

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    <artifactId>springboot-websocket-assigment</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>1.4.1.RELEASE</version>
+    </parent>
+    <properties>
+        <java.version>1.8</java.version>
```
Comment (manishmowall) [inline pom.xml:17]: second review comment

**Review by manishmowall — COMMENTED** [review]
submit comment

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-websocket</artifactId>
+        </dependency>
+
```
Comment (nitishm-webonise) [inline pom.xml]: first review comment

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        <dependency>
+            <groupId>org.webjars</groupId>
+            <artifactId>webjars-locator</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.webjars</groupId>
+            <artifactId>sockjs-client</artifactId>
+            <version>1.0.2</version>
```
Comment (nitishm-webonise) [inline pom.xml]: second review comment

**Comment by deeps-webonise** [comment]
first comment

**Pull request merged**